sysconfig/sysconfig.go

```diff
@@ -41,3 +43,9 @@ func ConfigureRunSystem(opts *Options) error {
 
 	return nil
 }
+
+// WritableDefaultsDir returns full path of subdir under
+// rootdir/_writable_defaults.
```

Comment:
// WritableDefaultsDir returns the full path of the joined subdir under the subtree for default content
// for system data living at rootdir,  i.e. rootdir/_writable_defaults/subdir...